src/web3/Client.ts

```diff
@@ -4,144 +4,198 @@ import { PrivateApiClient } from "./PrivateApiClient";
 import { PublicApiClient } from "./PublicApiClient";
 import { WalletClient } from "./WalletClient";
 import { SmartContractsClient } from "./SmartContractsClient";
-import { VaultClient } from "./VaultClient";
 import { IProvider, ProviderType } from "../interfaces/IProvider";
 import { DefaultProviderUrls, DefaultWsProviderUrls } from "./ClientFactory";
 import { IClient } from "../interfaces/IClient";
 import { WsSubscriptionClient } from "./WsSubscriptionClient";
 
+/**
+ * Maps and retrieves a websocket provider by a default provider
+ *
+ * @param provider - an enum of type `DefaultProviderUrls`
+ * @returns mapped `DefaultWsProviderUrls`
+ */
 export const getWsProvider = (provider: DefaultProviderUrls): DefaultWsProviderUrls => {
-	let wsProvider: DefaultWsProviderUrls;
-	switch (provider) {
-		case DefaultProviderUrls.LABNET: {
-			wsProvider = DefaultWsProviderUrls.LABNET;
-			break;
-		}
-		case DefaultProviderUrls.MAINNET: {
-			wsProvider = DefaultWsProviderUrls.MAINNET;
-			break;
-		}
-		case DefaultProviderUrls.LOCALNET: {
-			wsProvider = DefaultWsProviderUrls.LOCALNET;
-			break;
-		}
-		case DefaultProviderUrls.TESTNET: {
-			wsProvider = DefaultWsProviderUrls.TESTNET;
-			break;
-		}
-		default: {
-			wsProvider = DefaultWsProviderUrls.LOCALNET;
-		}
-	}
-	return wsProvider;
+  let wsProvider: DefaultWsProviderUrls;
+  switch (provider) {
+    case DefaultProviderUrls.LABNET: {
+      wsProvider = DefaultWsProviderUrls.LABNET;
+      break;
+    }
+    case DefaultProviderUrls.MAINNET: {
+      wsProvider = DefaultWsProviderUrls.MAINNET;
+      break;
+    }
+    case DefaultProviderUrls.LOCALNET: {
+      wsProvider = DefaultWsProviderUrls.LOCALNET;
+      break;
+    }
+    case DefaultProviderUrls.TESTNET: {
+      wsProvider = DefaultWsProviderUrls.TESTNET;
+      break;
+    }
+    default: {
+      wsProvider = DefaultWsProviderUrls.LOCALNET;
+    }
+  }
+  return wsProvider;
 };
 
-/** Massa Web3 Client wrapping all public, private, wallet and smart-contracts-related functionalities */
+
+/**
+ * Massa Web3 Client wrapping all public, private, wallet, websockets and smart-contracts-related functionalities 
+ *
+ */
 export class Client implements IClient {
+	/** @private */
```

Comment:
no need to specify private here I guess typedoc will take the visibility directly in the code https://typedoc.org/tags/private/